pytorch_lightning/plugins/precision/ipu.py

```diff
@@ -27,9 +27,20 @@
 
 
 class IPUPrecisionPlugin(PrecisionPlugin):
-    """Precision plugin for IPU integration."""
+    """Precision plugin for IPU integration.
+
+    Raises:
+        MisconfigurationException:
+            If the precision is neither 16 nor 32.
+    """
 
     def __init__(self, precision: int) -> None:
+        supported_precision_values = (16, 32)
+        if precision not in supported_precision_values:
+            raise MisconfigurationException(
```

Comment:
#10918 

```suggestion
            raise ValueError(
```

Reply:
Should we add a ImpactableConfigurationException? or use valueError?

Reply:
I think good old `ValueError` is good for now :)